packages/components/src/components/inline-message/inline-message.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { isMobile } from '@deriv/shared';
```

Comment:
@farzin-deriv please use `is_mobile` from ui store instead of this.

Reply:
@mahdiyeh-deriv This is inside `@deriv/components` package, We don't have access to stores here 🙇🏻